source/_docs/configuration/basic.markdown

```diff
@@ -1,117 +1,33 @@
 ---
 title: "Setup basic information"
 description: "Setting up the basic info of Home Assistant."
+related:
+- docs: /integrations/homeassistant/
+- docs: /docs/configuration/
```

Comment:
```suggestion
  - docs: /integrations/homeassistant/
  - docs: /docs/configuration/
```

Reply:
Thank you for the catch :fishing_pole_and_fish: :heart: